### PR TITLE
trimString-Helper Feature

### PR DESCRIPTION
**Pull request by svierling**
-helper to remove '#' from internal tag names

-trimString is registered and can be referenced, but appears to have no effect on string variables

-helper registered in default.hbs and called in custom-code-library.hbs

### CODE REVIEW

File: custom-code-library.hbs [hunk 1]
```diff
@@ -64,7 +64,7 @@
             <div>
                 {{#get "tags" limit="all"}}
                     {{#foreach tags visibility="internal"}}
-                        <h1>{{'name'}} Routines</h1>
+                        <h1>{{{trimString name}}} Routines</h1>
```
Comment (justinsantoro) [inline custom-code-library.hbs:67]: @orklepee where does `name` come from? Does it need those single quotes around it? I'm thinking there's probably another `name` defined somewhere and this one is getting clobbered which would explain why it silently "fails"

Comment (orklepee) [reply custom-code-library.hbs:67]: @justinsantoro `name` is an attribute of Ghost tags. It does not require single quotes around it, but it does work with and/or without them. Currently, if I remove "trimString" from the brackets, the internal tags are visible and their associated articles appear in dev.nolongerset/codelibrary/.

Comment (justinsantoro) [reply custom-code-library.hbs:67]: ok thanks.

File: default.hbs [hunk 2]
```diff
@@ -12,7 +12,13 @@
     {{!-- Theme assets - use the {asset} helper to reference styles & scripts,
     this will take care of caching and cache-busting automatically --}}
     <link rel="stylesheet" type="text/css" href="{{asset "built/screen.css"}}" />
-
+    <script src="https://twitter.github.io/typeahead.js/js/handlebars.js"></script>
+    <script>
+        Handlebars.registerHelper("trimString", function(passedString) {
+        var theString = passedString.substring(1,5);
+        return new Handlebars.SafeString(theString)
```
Comment (justinsantoro) [inline default.hbs:19]: can you stick some logging to the console in here?

```js
 Handlebars.registerHelper("trimString", function(passedString) {
 console.log("passed string: " + passedString);
 var theString = passedString.substring(1,5);
 console.log("theString: " + theString);
 return new Handlebars.SafeString(theString)
```

Comment (orklepee) [reply default.hbs:21]: @justinsantoro 6a6cb071cfd2068c737af8c8c428c023d2ec55b4

Comment (justinsantoro) [reply default.hbs:21]: what do we get

**Pull request merged**